### ISSUE TITLE
[nssdca-delivery] urn:nasa:pds:gbo.ast.catalina.survey::1.0 20220401

### ISSUE THREAD

**Issue by beatricemueller** [NASA-PDS/operations#276]
### Discipline Node Information

* **Delivering Node**: _please enter your DN here for record keeping purposes_
* PDS_PSI

* **NSSDCA Delivery Package**: _please upload the files output by PDS Deep Archive as a TAR.GZ or ZIP file here, or supply a URL to download from_
* https://sbnarchive.psi.edu/pds4/surveys/catalina_extras/sips/deltas/20220401_deltas.zip

* **Validation report**: _please upload a TXT report or screenshot of PDS4 Validate Tool run on your bundle. NSSDCA only accepts valid PDS4 bundles. be sure you run validate with the `-R pds4.bundle` flag enabled to ensure all integrity checks are completed successfully._
* https://sbnarchive.psi.edu/pds4/surveys/catalina_extras/validation/20220401.tar.gz

NOTE: If you have multiple delivery packages, we strongly encourage you to submit these in batches of 3 to 10 per issue with one ZIP file of the packages and another ZIP file of the validation reports. Please use a descriptive title, such as "Node Mission misc batch #".

----

### Engineering Node Process

See the internal EN process at https://pds-engineering.jpl.nasa.gov/content/nssdca_interface_process

**Comment by c-suh** [comment]
@beatricemueller this set has been posted for NSSDCA processing. From tomorrow, you can check the status at https://nssdc.gsfc.nasa.gov/psi/ReportPDS4.jsp using the SIP LID below:

SIP LID:
* urn:nasa:pds:system_bundle:product_sip_deep_archive:gbo.ast.catalina.survey_v1.0_20220401_delta_20220629164737571641

**Comment by smclaughlin7** [comment]
@beatricemueller @jstone-psi @c-suh The NSSDCA could not ingest 106 products in this SIP because the files were moved from the directories (URLs) specified in the SIP manifest *before* the NSSDCA could ingest (download):

SIP LID:

- urn:nasa:pds:system_bundle:product_sip_deep_archive:gbo.ast.catalina.survey_v1.0_20220401_delta_20220629164737571641::1.0

I will post a list of the products as soon as I receive it from our ingest team. 

All other products in this SIP were successfully ingested and are being archived to tape. Thanks!

**Comment by smclaughlin7** [comment]
@beatricemueller @jstone-psi @c-suh An additional remark from the NSSDCA's ingest of SIP LID urn:nasa:pds:system_bundle:product_sip_deep_archive:gbo.ast.catalina.survey_v1.0_20220401_delta_20220629164737571641::1.0:

This submission contained two collection inventory files that each listed several primary basic products which were not specified in the SIP manifest file and for which the actual data files do not exist online. This did not affect the NSSDCA ingest process which has successfully ingested the two collection products and their primary basic products as specified in the SIP manifest file.  However we want to inform PSI Node of our findings. We did not check newer versions of these two collection producs, where this very minor 'issue' may have already been resolved. The attached spread sheet identifies the two collection inventory files and their non-existent primary basic products.  Thanks!

[CSS_delta_SIP_20220401_Remark.xlsx](https://github.com/NASA-PDS/operations/files/9354265/CSS_delta_SIP_20220401_Remark.xlsx)
